openapi/nexus.json

```diff
@@ -7926,15 +8103,13 @@
             "$ref": "#/components/responses/Error"
           }
         }
-      }
-    },
-    "/v1/system/networking/switch-port-configuration/{configuration}": {
-      "get": {
+      },
+      "delete": {
         "tags": [
           "system/networking"
         ],
-        "summary": "Get information about a named set of switch-port-settings",
-        "operationId": "networking_switch_port_configuration_view",
+        "summary": "Delete switch port settings",
```

Comment:
settings or configuration?  (Fix in original code with docstring, check other places for consistency with new terminology)